doc/OnlineDocs/pyomo_modeling_components/Sets.rst

```diff
@@ -89,42 +139,44 @@ given, they are the first and last value in the sequence and the step
 size defaults to one. For example, the following declaration creates a
 set with the numbers 1.5, 5 and 8.5:
 
-.. literalinclude:: ../script_spy_files/Spy4Sets_RangeSet_simple_sequence.spy
-   :language: python
+.. doctest::
+   >>> model.G = RangeSet(1.5, 10, 3.5)
 
 Operations
 -----------
 
-Sets may also be created by assigning other Pyomo sets as in these
-examples that also illustrate the set operators union, intersection,
-difference, and exclusive-or:
-
-.. literalinclude:: ../script_spy_files/Spy4Sets_Set_operators.spy
-   :language: python
-
-The cross-product operator is the asterisk (*). For example, to assign a
-set the cross product of two other sets, one could use
+Sets may also be created by storing the result of *set operations* using
+other Pyomo sets.  Pyomo supports set operations including union, intersection,
+difference, and symmetric difference:
 
-.. literalinclude:: ../script_spy_files/Spy4Sets_Set_cross_product.spy
-   :language: python
+.. doctest::
+    model.I = model.A | model.D # union
```

Comment:
I think you're missing `>>>` in this code example and the other code examples below.